src/usePAPforms.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactElement, useState } from "react";
+import axios from "axios";
+
+export function usePAPforms(steps: ReactElement[]){
+    const [currentStepIndex, setCurrentStepIndex] = useState(0)
+
+    function submit(){
+       // let handleSubmit = async() => { try {axios.put('localhost:9999', FormData) } catch (err) { console.log(err); } };
+        axios.put('localhost:9998', FormData)
+        .then(response => {
+        console.log(response.data);
+        })
+        .catch(error => {
+        console.log(error);
```

Comment:
we need better error handling this, the website needs to display something, such as "There was a problem inputting your settings: